docs/scripts/buildApi.js

```diff
@@ -129,7 +129,9 @@ function buildDocs(options) {
 
   let reactAPI;
   try {
-    reactAPI = docgenParse(src);
+    reactAPI = docgenParse(src, null, null, {
+      filename: componentObject.filename,
```

Comment:
This is now recommended as of 4.0.0: https://github.com/reactjs/react-docgen/releases/tag/v4.0.0